transformation-engine/latest/config/index.md

```diff
@@ -8,7 +8,7 @@ The standalone Document Transformation Engine can be configured using the Web Co
 
 Use this information to configure the standalone Document Transformation Engine. You only need to change the password of the transformation service.
 
-1. Open your browser and navigate to `http://<transformation-host\>:<port\>/transformation-server/settings` or `https://` if you are using SSL.
+1. Open your browser and navigate to `http://<transformation-host>:<port>/transformation-backend/settings` or `https://` if you are using SSL.
```

Comment:
Is this correct, since it's part of configuration as opposed to just "using"?

Reply:
/transformation-server was correct since the user navigates to it. 

But to be entirely correct it needs to be http://<transformation-host\>:<port\>/transformation-server/#/settings for the new WebUI.